src/passes/cairoUtilImporter.ts

```diff
@@ -39,8 +41,11 @@ export class CairoUtilImporter extends ASTMapper {
   }
 
   visitElementaryTypeName(node: ElementaryTypeName, ast: AST): void {
-    if (primitiveTypeToCairo(node.name) === 'Uint256') {
+    const cairoType = primitiveTypeToCairo(node.name);
+    if (cairoType === 'Uint256') {
```

Comment:
The change will cause merge conflicts with #967

Reply:
yeah, will resolve conflicts if that PR goes first